frontend/javascripts/admin/api/jobs.ts

```diff
@@ -0,0 +1,346 @@
+  jobURLPath: string,
+  organizationName: string,
+  datasetName: string,
+  fallbackLayerName: string,
+  volumeLayerName: string | null | undefined,
+  newDatasetName: string,
+  annotationId: string,
+  annotationType: APIAnnotationType,
+  outputSegmentationLayerName?: string,
+  mergeSegments?: boolean,
+): Promise<APIJob> {
+  const requestURL = new URL(
+    `/api/jobs/run/${jobURLPath}/${organizationName}/${datasetName}`,
+    // @ts-expect-error ts-migrate(2345) FIXME: Argument of type 'Location' is not assignable to parameter of type 'string | URL | undefined'.
+    location,
+  );
+  if (volumeLayerName != null) {
+    requestURL.searchParams.append("volumeLayerName", volumeLayerName);
+  }
+  requestURL.searchParams.append("fallbackLayerName", fallbackLayerName);
+  requestURL.searchParams.append("annotationId", annotationId);
+  requestURL.searchParams.append("annotationType", annotationType);
+  requestURL.searchParams.append("newDatasetName", newDatasetName);
+  if (outputSegmentationLayerName != null) {
+    requestURL.searchParams.append("outputSegmentationLayerName", outputSegmentationLayerName);
+  }
+  if (mergeSegments != null) {
+    requestURL.searchParams.append("mergeSegments", mergeSegments.toString());
+  }
+  return Request.receiveJSON(requestURL.href, {
+    method: "POST",
+  });
+}
+
+export function startMaterializingVolumeAnnotationJob(
+  organizationName: string,
+  datasetName: string,
+  fallbackLayerName: string,
+  volumeLayerName: string | null | undefined,
+  newDatasetName: string,
+  outputSegmentationLayerName: string,
+  annotationId: string,
+  annotationType: APIAnnotationType,
+  mergeSegments: boolean,
+): Promise<APIJob> {
+  return startSegmentationAnnotationDependentJob(
+    "materializeVolumeAnnotation",
+    organizationName,
+    datasetName,
+    fallbackLayerName,
+    volumeLayerName,
+    newDatasetName,
+    annotationId,
+    annotationType,
+    outputSegmentationLayerName,
+    mergeSegments,
+  );
+}
+
+export function startMitochondriaInferralJob(
+  organizationName: string,
+  datasetName: string,
+  layerName: string,
+  bbox: Vector6,
+  outputSegmentationLayerName: string,
+  newDatasetName: string,
+): Promise<APIJob> {
+  const urlParams = new URLSearchParams({
+    layerName,
+    bbox: bbox.join(","),
+    outputSegmentationLayerName,
+    newDatasetName,
+  });
+  return Request.receiveJSON(
+    `/api/jobs/run/inferMitochondria/${organizationName}/${datasetName}?${urlParams.toString()}`,
+    {
+      method: "POST",
+    },
+  );
+}
+type AiModelCategory = "em_neurons" | "em_nuclei";
+
+type AiModelTrainingAnnotationSpecification = {
+  annotationId: string;
+  colorLayerName: string;
+  segmentationLayerName: string;
+  mag: Vector3;
+};
+
+type RunTrainingParameters = {
+  trainingAnnotations: Array<AiModelTrainingAnnotationSpecification>;
+  name: string;
+  comment?: string;
+  aiModelCategory?: AiModelCategory;
+  workflowYaml?: string;
+};
```

Comment:
Since these are not exported currently, I'd like to keep them here. I'm always glad when I see definitions that are local to the module because it's less mental overhead :)